task/sbom-json-check/0.1/README.md

```diff
@@ -0,0 +1,18 @@
+# sbom-json-check task
+
+## Description:
+Check the syntax of the sbom-cyclonedx.json file which should be found in the /root/buildinfo/content_manifests/ directory
```

Comment:
I think we can also explain here what SBOM (Software Bill of Materials) is - [this Red Hat post](https://www.redhat.com/en/blog/how-red-hat-addressing-demand-develop-offerings-more-securely) could be a helpful reference, though we could also link to the [official CISA article](https://www.cisa.gov/sbom).
Feel free to offer other suggestions of documentation links here too.

Reply:
@dheerajodha is addressing this [here](https://github.com/redhat-appstudio/build-definitions/pull/352/files#diff-605867080648772a6b7591b4b5d914e3362891832c9099ed5cbb2a4b0969aa3e)